traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -312,6 +312,9 @@ sub param_data {
 		if ( $row->parameter->name eq "location" ) {
```

Comment:
The parameters are going to need to be overhauled to make it compatible with other CDN software. I'm planning on working on that sort of thing next. 

Regarding scope, I didn't want to rely on straight code. However, we may be able to do so if it's a significant concern. The get_scope function is there as a fallback in case a config file doesn't have a scope, but thinking about it now it's entirely possible we could do away with the parameter altogether. The original thought was that for anything not already defined I wanted to be able to define it in the db so we could add config files without code.  But i think all those files would be take and bake generated anyway, which is server based and the default return when a file name isn't matched. So... I think we're really just dealing with a single set of core files. 

I'll look into it.  I think you might be correct.